### DISCUSSION TITLE
EJBCA Enterprise pricing

### DISCUSSION THREAD

**Discussion by danieldownload** [Keyfactor/ejbca-ce#434] [General]
Hi there,

I'm looking for an alternative for our MS CA for a while now. Most important point is to rollout computer certs on Windows clients via MS GPO. As far as I have seen, the only way to do it like a MS CA does it, is to use ejbca enterprise.

Does anybody have a rough price info for the enterprise edition?

Or is there another way to automatically rollout and renew computer/client certs in connection with ejbca-ce?

Thx a lot,

Daniel

**Comment by primetomas** [comment]
There are many ways to rollout certificates, mostly homegrown. But Windows auto enrollment is as you say typically the best one in a Windows environment.

Enterprise pricing depend on several factors, such as scale of deployment, deployment model, etc. Feel free to dm me and I can connect you with the right person.

Cheers,
Tomas
